### DISCUSSION TITLE
Documentation Styling Icons: --fa-bounce-height redefinition?

### DISCUSSION THREAD

**Discussion by thekingofspain** [FortAwesome/Font-Awesome#20163] [General]
In the previous FontAwesome documentation around styling icons, --fa-bounce-height had an "Accepted Values" value of "Any valid CSS height value".

In the new FontAwesome documentation the "Accepted Values" value seemed to change to "Any valid CSS number".

Is this an error in the updated documentation, or 
did the functionality change in FontAwesome, or 
did I make a mistake(referencing or a bad assumption) when implemented my code on the previous documentation?

thanks

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

I think this may be an inaccuracy in the documentation, because "number" is not correct but also "height" may not be 100% correct.

As per https://developer.mozilla.org/en-US/docs/Web/CSS/height, we have `length`, `percentage`, `keyword`, and `global` values.

In this context, this value appears to accept `length` and `percentage` values only

The most accurate description should be `length` and `percentage` values:
- https://developer.mozilla.org/en-US/docs/Web/CSS/length
- https://developer.mozilla.org/en-US/docs/Web/CSS/percentage

**NOTE:** This should also be true for `--fa-bounce-rebound`

/cc @robmadole
